prov/efa/src/efa_errno.h

```diff
@@ -55,22 +55,22 @@
  * These status codes correspond directly to the `efa_io_comp_status` codes
  * defined by the RDMA Core EFA provider.
  */
-#define EFA_IO_COMP_STATUSES(_)									\
-	_(0,	OK,				Success)					\
-	_(1,	FLUSHED,			Flushed during queue pair destroy)		\
-	_(2,	LOCAL_ERROR_QP_INTERNAL_ERROR,	Internal queue pair error)			\
-	_(3,	LOCAL_ERROR_INVALID_OP_TYPE,	Invalid operation type)				\
-	_(4,	LOCAL_ERROR_INVALID_AH,		Invalid address handle (local))			\
-	_(5,	LOCAL_ERROR_INVALID_LKEY,	Invalid local key (LKEY))			\
-	_(6,	LOCAL_ERROR_BAD_LENGTH,		Message too long)				\
-	_(7,	REMOTE_ERROR_BAD_ADDRESS,	RKEY not registered or does not match remote IOVA)	\
-	_(8,	REMOTE_ERROR_ABORT,		Receiver connection aborted)			\
-	_(9,	REMOTE_ERROR_BAD_DEST_QPN,	Invalid receiver queue pair number (QPN))	\
-	_(10,	REMOTE_ERROR_RNR,		Receiver not ready)				\
-	_(11,	REMOTE_ERROR_BAD_LENGTH,	Receiver scatter-gather list (SGL) too short)	\
-	_(12,	REMOTE_ERROR_BAD_STATUS,	Unexpected status received from remote)		\
-	_(13,	LOCAL_ERROR_UNRESP_REMOTE,	Unresponsive receiver (connection never established or unknown))	\
-	_(14,	REMOTE_ERROR_UNKNOWN_PEER,	Invalid address handle on remote)
+#define EFA_IO_COMP_STATUSES(_)														\
```

Comment:
Overall I imagine it would be cleaner to use well-known acronyms, which should benefit the user in terms of concise logging.

For example:
- Queue pair: QP
- Local key: LKEY
- Remote key: RKEY
- Address handle: AH
- Work queue entry: WQE

Reply:
Updated

Reply:
I strongly disagree with using only acronyms/abbreviations other than very ubiquitous ones, e.g. `ASCII` or `I/O`. `Word/phrase (acronym/abbreviation)` is ideal; the error message strings are supposed to be short enough anyway.

Reply:
I can get the full name back while keeping the acronyms in parenthesis

Reply:
@darrylabbate Are you concerned with ambiguity of the acronyms or something else? In the context of libfabric and rdma those acronyms are common enough that I doubt they require more annotations.

Reply:
Last push got full names back. I selectively add acronyms for some error code because I don't want to have too many parenthesis.